situp-plugins/common/src/test/java/com/amazon/situp/plugins/processor/state/ProcessorStateTest.java

```diff
@@ -1,20 +1,16 @@
 package com.amazon.situp.plugins.processor.state;
 
 import com.amazon.situp.processor.state.ProcessorState;
-import java.io.Serializable;
-import java.nio.charset.StandardCharsets;
-import java.util.Collection;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Random;
-import java.util.UUID;
-import java.util.function.BiFunction;
-import java.util.stream.Collectors;
 import org.junit.After;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
 
+import java.io.Serializable;
+import java.util.*;
```

Comment:
nit: Avoid *.

Reply:
oops, yeah that was auto import. I will remove it